src/StarsRatingBar/test/StarsRatingBar.spec.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { createRendererWithUniDriver, cleanup } from '../../../test/utils/unit';
+
+import StarsRatingBar from '../StarsRatingBar';
+import { starsRatingBarPrivateDriverFactory } from './StarsRatingBar.private.uni.driver';
+
+describe(StarsRatingBar.displayName, () => {
+  const render = createRendererWithUniDriver(
+    starsRatingBarPrivateDriverFactory,
+  );
+
+  afterEach(() => {
+    cleanup();
+  });
```

Comment:
Eliminate unnecessary scope

```suggestion
  afterEach(cleanup);
```

Reply:
This code came from the template so maybe we should change the template as well.